### PR TITLE
207 SIM-e2e tests for validators configuration

### PR DESCRIPTION
**Pull request by bradleySuira**

### CODE REVIEW

File: frontend/src/views/Simulator/SettingsView.vue [hunk 1]
```diff
@@ -65,7 +65,8 @@ async function handleDeleteValidator() {
     notify({
       title: 'OK',
       text: 'Validator deleted successfully',
-      type: 'success'
+      type: 'success',
+      duration: 53000
```
Comment (cristiam86) [inline frontend/src/views/Simulator/SettingsView.vue:69]: @bradleySuira is this on milliseconds? why 53000?

File: frontend/test/pages/SettingsPage.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  async openNewValidatorModal() {
+    const locator = By.xpath(`//button[contains(text(), 'New Validator')]`)
+    return this.driver.wait(until.elementLocated(locator), 5000).click()
+  }
+  async getValidatorsElements(): Promise<WebElement[]> {
+    return this.driver.findElements(By.xpath("//div[@data-testid = 'validator-item-container']"))
+  }
+
+  async createValidator() {
```
Comment (cristiam86) [inline frontend/test/pages/SettingsPage.ts:16]: @bradleySuira this method should receive the provider, model, and stake so it is more reliable when testing the update validator

File: frontend/vite.config.ts.timestamp-1719509065635-6c2a7c57083b5.mjs [hunk 3]
```diff
@@ -0,0 +1,36 @@
+// vite.config.ts
```
Comment (cristiam86) [inline frontend/vite.config.ts.timestamp-1719509065635-6c2a7c57083b5.mjs:1]: @bradleySuira should this file be commited?

**Review by cristiam86 — CHANGES_REQUESTED** [review]

File: frontend/src/views/Simulator/SettingsView.vue (hunk 1 above)
Comment (bradleySuira) [reply frontend/src/views/Simulator/SettingsView.vue:69]: fixed, removed the duration

File: frontend/vite.config.ts.timestamp-1719509065635-6c2a7c57083b5.mjs (hunk 3 above)
Comment (bradleySuira) [reply frontend/vite.config.ts.timestamp-1719509065635-6c2a7c57083b5.mjs:1]: removed and added to ignore files

File: frontend/test/pages/SettingsPage.ts (hunk 2 above)
Comment (bradleySuira) [reply frontend/test/pages/SettingsPage.ts]: fixed now 
we have all params in the util method
```
 async createValidator({
    provider,
    model,
    stake
  }: {
    provider: string
    model: string
    stake: number
  })
```

**Review by cristiam86 — APPROVED** [review]

**Pull request merged**